### PR TITLE
Featurs & bugfixes

### PR DESCRIPTION
**Pull request by ParadiseFallen**
Bugfix:
- Fixed ru / jp generation of docs. Now you get rranslated page. (was en only 
Features:
- Ru translation. Was translated /animations. [WIP]

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/stride3d/stride-docs?pullRequest=88) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/stride3d/stride-docs?pullRequest=88) before we can accept your contribution.<br/><br/>:x: ParadiseFallen [sign now](https://cla.dotnetfoundation.org/stride3d/stride-docs?pullRequest=88)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/stride3d/stride-docs?pullRequest=88) it.</sub>

**Comment by VaclavElias** [comment]
I will review this.

File: ru-build.ps1 [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+(Get-Content ru_tmp/docfx.json) -replace '"_enableSearch": true','"_enableSearch": false' | Set-Content ru_tmp/docfx.json
+deps\docfx\docfx.exe build ru_tmp\docfx.json
+Remove-Item ru_tmp -recurse 
+if ($LastExitCode -ne 0)
+{
+	Write-Host "Failed to build doc"
+	exit $LastExitCode
+}
+
+Write-Host "Japanese documentation built."
```
Comment (VaclavElias) [inline ru-build.ps1:48]: This should be "Russian documentation built."

**Comment by VaclavElias** [comment]
I have tested this locally and all works.

Note, that this PR as mentioned above:

1. Is fixing JP pages showing up
   - Currently on the live website you can see JP only left navigation, when you browse pages they are in EN 
   - When the **jp_tmp** folder is created, there are JP pages, but when you run **docfx.exe build ...**, it somehow uses EN pages
   - I tried a few past versions of docs back to April 28 2022 and they had the same issue, no idea when it stopped working
   - The issue is "resolved" by changing ```_enableSearch=false```, this will build JP pages, but the top search bar will stop working. I think this is better solution, than not to show up JP pages at all. I will open separate Issue, which is mentioning _enableSearch issue
   - _enableSearch is also used in the docfx.exe, so maybe there is some bug in their code base..
2. Adds RU pages, they are all working, @ParadiseFallen is planning to update all of them to RU. Currently the navigation and animation folder is translated as you can see in PR.

**Review by VaclavElias — COMMENTED** [review]
All seems ok

**Comment by VaclavElias** [comment]
@Aggror, @xen2 I will leave it with you now. I collected all information needed to update docs "How to add another language to stride-docs".

**Comment by Aggror** [comment]
Good work @ParadiseFallen and @VaclavElias.

**Comment by manio143** [comment]
@ParadiseFallen What is the status of this PR? I see some WIP tags in the commits, so want to understand if this is ready to be merged or if there's still more work to be done.

**Comment by ParadiseFallen** [comment]
@manio143 as we decide in discord there is no reason to make translations and for now its enough en docs (support issues)

**Comment by VaclavElias** [comment]
Closing this PR as this needs to be re-done but we are thinking to not support multiple languages at the moment. The translated content could be re-used if we support multiple languages in the future.

**Comment by ParadiseFallen** [comment]
for now i also think that we need to support `ONLY` en version of docs

**Pull request closed without merging**